src/item.cpp

```diff
@@ -3290,11 +3279,6 @@ bool item::destroyed_at_zero_charges() const
     return (is_ammo() || is_food());
 }
 
-bool item::is_var_veh_part() const
```

Comment:
You should also remove the function declaration when you remove a function definition. People might use the declared function and will get obscure linker errors.